Solutions/Onapsis Defend: Integrate Unmatched SAP Threat Detection & Intel with Microsoft Sentinel/Data Connectors/Onapsis.json

```diff
@@ -80,15 +80,15 @@
                 {
                     "parameters": {
                         "label": "Deploy push connector resources",
-                        "applicationDisplayName": "Onapsis Integration push to Microsoft Sentinel"
+                        "applicationDisplayName": "Onapsis Defend Integration push to Microsoft Sentinel"
                     },
                     "type": "DeployPushConnectorButton_test"
                 }
             ]
         },
         {
-            "title": "2. Maintain the data collection endpoint details and authentication info in Onapsis Integration",
-            "description": "Share the data collection endpoint URL and authentication info with the Onapsis Integration administrator to configure the Onapsis Integration to send data to the data collection endpoint.\n\nLearn more from [this blog series](https://community.Onapsis.com/t5/enterprise-resource-planning-blog-posts-by-members/ultimate-blog-series-Onapsis-logserv-integration-with-microsoft-sentinel/ba-p/14126401).",
+            "title": "2. Maintain the data collection endpoint details and authentication info in Onapsis Defend Integration",
+            "description": "Share the data collection endpoint URL and authentication info with the Onapsis Defend Integration administrator to configure the Onapsis Defend Integration to send data to the data collection endpoint.\n\nLearn more from [this blog series](https://community.Onapsis.com/t5/enterprise-resource-planning-blog-posts-by-members/ultimate-blog-series-Onapsis-logserv-integration-with-microsoft-sentinel/ba-p/14126401).",
```

Comment:
the blog ref points still at the sap logserv solution. It should be a public onapsis doc rgearding setup or similar